src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java

```diff
@@ -204,5 +208,19 @@ public void windowsRawMessage() {
         assertEquals(Type.raw, unifiedMessage.getMessage().getObject().getWindows().getType());
     }
 
+    @Test
+    public void windowsAndIosCombinedMessage() {
+        UnifiedMessage unifiedMessage = UnifiedMessage.withMessage()
+                .windows()
+                  .raw()
+                  .build()
+                .apns()
+                  .contentAvailable()
+                  .build()
+                .build();
+        assertEquals(Type.raw, unifiedMessage.getMessage().getObject().getWindows().getType());
+        assertEquals(true, unifiedMessage.getMessage().getObject().getApns().isContentAvailable());
```

Comment:
let's, at some point, get rid of that and use AssertJ instead  - but that should be a different PR